Resources/public/css/theme/views/serverside.css

```diff
@@ -14,3 +13,56 @@
 .ez-view-serversideview .ez-tabs.ez-serverside-content {
     border: 0;
 }
+
+.ez-view-serversideview .ez-contenttype-view {
+    background: #FAFAFA;
+}
+
+.ez-view-serversideview .ez-contenttype-view .ez-contenttype-title,
+.ez-view-serversideview .ez-contenttype-view .ez-contenttype-subtitle {
+    background: transparent;
+    color: #2b84b1;
+    border: none;
+    border-bottom: 1px solid #2b84b1;
+}
+
+.ez-view-serversideview .ez-contenttype-view .ez-contenttype-metadata {
+    background: #f3f3f3;
+}
+
+.ez-view-serversideview .ez-contenttype-view .ez-contenttype-metadata .ez-contenttype-metadata-name {
+    background: #ededed;
+    font-weight: bold;
+}
+
+.ez-view-serversideview .ez-contenttype-view .ez-contenttype-metadata .ez-contenttype-metadata-name .help {
+    font-weight: normal;
+    font-style: italic;
+    font-size: 0.7em;
+}
+
+.ez-view-serversideview .ez-contenttype-view .ez-field-definitions .ez-field-definition-header {
+    background: #c2c2c2;
+}
+
+.ez-view-serversideview .ez-contenttype-view .ez-field-definitions .ez-field-definition-main-properties {
+    border-top: 1px solid #c2c2c2;
+}
+
+.ez-view-serversideview .ez-contenttype-view .ez-field-definitions .ez-field-definition-properties .ez-field-definition-property .ez-field-definition-property-name {
```

Comment:
you don't need to be that specific, I think you can omit `.ez-field-definitions .ez-field-definition-properties .ez-field-definition-property` (and more or less the same can be applied to others selector)